build/build-bundle.js

```diff
@@ -121,6 +121,7 @@ function minifyScript(filePath) {
     shouldPrintComment: () => false, // Don't include @license or @preserve comments either
     plugins: [
       'syntax-object-rest-spread',
+      'syntax-async-generators',
```

Comment:
this change was needed for supporting `for await...of` in our client minifying pipeline. It *feels* like we shouldn't need the plugin now that it's stage 4 and shipping everywhere, but I don't care enough to look into babel more :)

note that this isn't transpiling anything (we only minify), it's just so the babel parser understands the syntax and doesn't throw when seeing this.

(no idea if the `brfs` change is needed, it's just that 10 is the latest version, so might as well)